### ISSUE TITLE
Add an option to change column header background color

### ISSUE THREAD

**Issue by krupenja** [mhutchie/vscode-git-graph#53]
**Describe the feature that you'd like**
I would like to have an option to change column header background color.

**Additional context (optional)**
This is especially important for completely flat themes, like [Material Theme](https://marketplace.visualstudio.com/items?itemName=Equinusocio.vsc-material-theme). Note how on the screenshot the column headers are the only element which is light grey. Having an option to change it dark grey like most other UI elements would be great.

![image](https://user-images.githubusercontent.com/26354715/56533454-5511b380-6560-11e9-8e22-a39ef631a127.png)

**Comment by mhutchie** [comment]
Thanks for the suggestion, this would be nice to have!

**Comment by mhutchie** [comment]
I decided that the background colour of the control bar and column headers wasn't necessary, so I removed it and made a few tweaks to the borders to work better with no background. I think the control bar and column headers look much better than before.

This will be available in v1.4.6, which will be released in the next few hours.

I've tested these changes with the theme that you mentioned, and I don't think adding a new setting is necessary any more. Once you've tried the changes in v1.4.6, let me know if you still think the setting is needed.
